script.embuary.info/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.embuary.info" name="Embuary Info" version="1.0.7" provider-name="sualfred">
+	<requires>
+		<import addon="xbmc.python" version="2.26.0"/>
+		<import addon="script.module.requests" version="2.9.1"/>
+		<import addon="script.module.arrow" version="0.10.0"/>
+		<import addon="script.module.simplecache" version="1.0.17" />
+	</requires>
+	<extension point="xbmc.python.script" library="default.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Embuary Info Script</summary>
+		<description lang="en_GB">Helper script for skinner to provide The Movie DB informations for persons, shows and movies.</description>
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE Version 2, June 1991</license>
```

Comment:
Personally I prefer the spec license identifier

Reply:
Spdx

Reply:
changed